src/pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -185,7 +185,7 @@ def __init__(
         self._layer_sync: Optional[LayerSync] = NativeSyncBatchNorm() if sync_batchnorm else None
         self.checkpoint_io: Optional[CheckpointIO] = None
         self._amp_type_flag: Optional[LightningEnum] = None
-        self._amp_level_flag: Optional[str] = amp_level
+        self._amp_level_flag: str = amp_level
```

Comment:
Should this stay `Optional` because of `amp_level: Optional[str] = None`, no?
```suggestion
        self._amp_level_flag: str = amp_level
```

Reply:
oh yes. forgot to revert the typing